### PR TITLE
add get MTA ID cmd

### PR DESCRIPTION
**Pull request by rayakoren**
Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: cmd/mta.go [hunk 1]
```diff
@@ -165,3 +168,19 @@ var updateBuildParametersCmd = &cobra.Command{
 	SilenceUsage:  true,
 	SilenceErrors: true,
 }
+
+// getMtaIdCmd - get MTA ID.
+var getMtaIdCmd = &cobra.Command{
+	Use:   "mtaId",
```
Comment (tal-sapan) [inline cmd/mta.go:174]: Call the command `id` instead of `mtaId` (it will look like `mta get id`)

File: mta/mta_services.go [hunk 2]
```diff
@@ -3,15 +3,15 @@ package mta
...
 	"crypto/sha1"
 	"fmt"
-	"gopkg.in/yaml.v3"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"gopkg.in/yaml.v3"
```
Comment (tal-sapan) [inline mta/mta_services.go:12]: Why did you move it to a separate import group?

Comment (rayakoren) [reply mta/mta_services.go:12]: I didn't... when I save it moves by itself ... I'll put it back
